lib/virtfs/block_io.rb

```diff
@@ -11,6 +11,8 @@ module VirtFS
   class BlockIO
     MIN_BLOCKS_TO_CACHE = 64
 
+    attr_reader :seek_pos
```

Comment:
@movitto Remember, this is implementing an existing, functionally complete, interface. When I say existing, I don't only mean in MIQ, but for general block IO. The FS code should be written as though it were reading directly from a block device, where seek_pos is not available.

Also, relying on the underlying device's position would make it very difficult to support thread concurrency in the future. The maintenance of read position should be maintained in the FS code itself because that's the level at which the locking scheme would be applied.